test2.rb

```diff
@@ -0,0 +1,26 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
+
+ l = lambda {|a| a }
+ l.(x)
```

Comment:
__[development][rubocop]__
* Inconsistent indentation detected.
* Prefer the use of `lambda.call(...)` over `lambda.(...)`. [:ref](https://github.com/bbatsov/ruby-style-guide#proc-call)
* Shadowing outer local variable - `a`.

<!-- Comment by [SideCI](https://sideci.com) -->